examples/benchmark_xgboost.py

```diff
@@ -0,0 +1,1100 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 15:53:27 2016
+
```

Comment:
@miaecle This the exact script I used to generate benchmark scores only for XGBoost models. I run 3 split methods on datasets:
 'tox21', 'sider', 'muv', 'toxcast', 'clintox', 'hiv', 'sampl', 'delaney', 'nci', 'pdbbind'
and index split on dataset:
'PCBA'.

My intention is that you can decide which dataset you want to include in the official benchmark.py scirpt.